.travis.yml

```diff
@@ -79,6 +79,22 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - rust: stable
+     env:
+      NGINX_VER=1.16.1
+     before_install:
+      # Install and use the current stable release of Go
+      - gimme --list
+      - eval "$(gimme stable)"
+      - gimme --list
+     script:
+      - curl -O https://nginx.org/download/nginx-$NGINX_VER.tar.gz
+      - tar xvf nginx-$NGINX_VER.tar.gz
+      - |
+        cd nginx-$NGINX_VER &&
+        patch -p01 < ../extras/nginx/nginx-1.16.patch &&
+        ./configure --with-http_ssl_module --with-http_v2_module --with-http_v3_module --with-openssl="../deps/boringssl" --with-quiche=".." &&
+        make -j`nproc`
```

Comment:
Do we add a sample `nginx.conf`? If so I think it's good to run `nginx -c nginx-quic.conf -t` at the end, just for sanity check.

Reply:
Maybe in the future we'll probably want to run the HTTP/3 tests, but for now I just want to make sure this builds as a start.

Reply:
`nginx -t` will check configuration is valid, not serving anything (exit immediately). If new patch breaks anything config related it will detect.

Reply:
Yeah but we don't have the configuration as file, so once we set that up we might as well run the tests as well.